.github/workflows/test.yml

```
@@ -75,3 +75,17 @@ jobs:
         uses: ./
         with:
           results-file: build_error.txt
+
+      - name: Test no-summary case (no "test result: ..." line)
```

actionlint (1.7.10)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)